themes/beaver/layouts/partials/blog/footer.html

```diff
@@ -0,0 +1,58 @@
+</div>
```

Comment:
is it correct?

Reply:
In hugo you can do both way.

Converted posts and post list to use baseof

Reply:
This request is aimed at ensuring that developers do not introduce bugs while editing this file, as it reduces hard coupling with some parent context.